### DISCUSSION TITLE
HTTP 403 copilot jetbrains

### DISCUSSION THREAD

**Discussion by Sirnilin** [community/community#139276] [GitHub Education]
### Hi everyone,

Hello!

I have a problem when using Copilot in the JetBrains\app.  I get a 403 HTTP error when I try to write a message via Copilot. I'm in a region where access to Copilot may be restricted, so I use the\proxy.

Thank you for your time and attention to my problem!
![image](https://github.com/user-attachments/assets/6137b202-c616-430d-8787-50508f5fc71e)

**Comment by sam-math** [comment]
Is it only specific to jetbrains? If so it might be about the proxy config. Check this: [https://docs.github.com/en/copilot/managing-copilot/configure-personal-settings/configuring-network-settings-for-github-copilot?tool=jetbrains](https://docs.github.com/en/copilot/managing-copilot/configure-personal-settings/configuring-network-settings-for-github-copilot?tool=jetbrains)

I'd also check with other proxies.

If not, 403 might be connected to authentication (check if you're properly logged in)

**Reply by Sirnilin** [reply]
I've been using Copilot for a few months now and there were no problems. But recently I got this problem. I tried different proxies and VPNs, but nothing helps, I still get 403.
